Samples/SimpleMvcExample.Messages/BackupDatabaseServer.cs

```diff
@@ -10,6 +11,7 @@ public class BackupDatabaseServer : ICommand
         
         [DisplayName("Destination Filename")]
         [Description("Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vel turpis vel diam dignissim euismod ut non velit. Integer eget.")]
+        [StringPropertySuggestions(ResourceProviderName = "FilenameSuggestionProvider")]
         public string? DestinationFilename { get; init; }
```

Comment:
I'm wondering if we should be building out a better way to describe our data types on command properties for the UI to generate. Some attributes like Custom Fields where they define the data type.

Something like

```
  [DataType(Options), OptionsValueProvider(ResourceProviderName = "FilenameSuggestionProvider")]
  public string? DestinationFilename { get; init; }

  [DataType(Text), RegexFormat("DFN-[0-9]{4}")]
  public string TicketNumber{ get; init; }
```

Reply:
I think that is a good idea, this attribute is purely to specify a list of suggested values for a free text input, such as a database name, the user is still free to enter anything they like.